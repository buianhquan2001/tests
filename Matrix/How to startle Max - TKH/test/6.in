24
62
93
109
85
86
118
83
107
71
84
118
72
116
109
120
94
64
99
95
64
104
90
95
120
