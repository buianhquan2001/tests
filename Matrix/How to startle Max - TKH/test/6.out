217689433
482361088
612955947
100543565
980141720
904799209
677424793
472606716
51288244
130003129
904799209
170325148
670858904
612955947
951234175
806075195
810370359
623131616
398885785
810370359
391522634
2991045
398885785
951234175
